source/blog/2019-01-04-the-ember-times-issue-79.md

```diff
@@ -55,7 +55,11 @@ A **new RFC** proposes the [deprecation of these abundant properties](https://gi
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [API Search Performance](#api-search-performance)
+
+<img class="float-left small transparent padded" alt="Algolia search improvements" src="/images/blog/emberjstimes/algolia-search-performance-improvements.png" />
+
+Our friends are Algolia have enabled us to improve API docs search performance by as high as 285% for some countries in Europe! The map shows data centers where the API docs are currently stored and (on the right) the performance improvement for the top searching countries.
```

Comment:
I think it would be cool to add a sentence or two for new readers on how Algolia helps us in general and where on our website it's used right now. What do you think about

```suggestion
The keyword search for both the [Ember Guides](https://github.com/ember-learn/guides-app) and the [Ember API Docs](https://github.com/ember-learn/ember-api-docs) is powered by [algoliasearch](https://github.com/algolia/algoliasearch-client-javascript) and has now become even more performant!

Our friends at Algolia have enabled us to improve API docs search performance by as high as 285% for some countries in Europe! The map shows data centers where the API docs are currently stored and (on the right) the performance improvement for the top searching countries.
```

Reply:
good call 👍 ! Added a bit of explanation, thanks for this